package.json

```diff
@@ -32,11 +32,13 @@
     "@types/jasmine": "~4.0.0",
     "autoprefixer": "^10.4.13",
     "jasmine-core": "~4.3.0",
+    "json-server": "^0.17.1",
     "karma": "~6.4.0",
     "karma-chrome-launcher": "~3.1.0",
     "karma-coverage": "~2.2.0",
     "karma-jasmine": "~5.1.0",
     "karma-jasmine-html-reporter": "~2.0.0",
+    "nodemon": "^2.0.20",
```

Comment:
is this useful?

Reply:
This is part of the fake database. Def dependency only